README.md

```diff
@@ -408,17 +408,11 @@ and CUDNN7 installations on your machine (for example, preinstalled on your
 cloud VM), you can run
 
 ```bash
-# install jaxlib
-PYTHON_VERSION=cp37  # alternatives: cp36, cp37, cp38
-CUDA_VERSION=cuda100  # alternatives: cuda100, cuda101, cuda102, cuda110
-PLATFORM=manylinux2010_x86_64  # alternatives: manylinux2010_x86_64
-BASE_URL='https://storage.googleapis.com/jax-releases'
-pip install --upgrade $BASE_URL/$CUDA_VERSION/jaxlib-0.1.52-$PYTHON_VERSION-none-$PLATFORM.whl
-
-pip install --upgrade jax  # install jax
+pip install --upgrade pip
+pip install --upgrade jax jaxlib==0.1.52+cuda110 -f https://storage.googleapis.com/jax-releases/jax_releases.html
 ```
 
-The library package name must correspond to the version of the existing CUDA
+The jaxlib version must correspond to the version of the existing CUDA
 installation you want to use, with `cuda110` for CUDA 11.0, `cuda102` for CUDA
 10.2, `cuda101` for CUDA 10.1, and `cuda100` for CUDA 10.0. To find your CUDA
 and CUDNN versions, you can run commands like these, depending on your CUDNN
```

Comment:
This paragraph doesn't say why you need to know your cuDNN version, so I'm not sure it needs to say how to find it?

Reply:
You're right, A+ reading comprehension! I removed the cuDNN bits.